burotel/indico_burotel/client/styles/palette.scss

```diff
@@ -14,5 +14,5 @@ $dark-highlight-color: #b1604b;
 $primary-color: #2185d0;
 
 %rb-splash-background {
-    background-image: url('burotel:images/splash.png');
+    background-image: url('burotel:images/splash.jpeg');
```

Comment:
.jpg - jpeg is a very uncommon extension even if that's the name of the format ;)